### ISSUE TITLE
Unzipping helm no longer works

### ISSUE THREAD

**Issue by KidkArolis** [streamplace/npm-kubetools#3]
It creates a blank `node_modules/helm-cli/helm` file

**Comment by iameli** [comment]
Yeah, I've personally stopped using this project for anything other than `kubectl` unfortunately. Not 100% sure what's up, possible the Helm folks changed the format of their tarball
